src/main.rs

```diff
@@ -164,10 +161,24 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
             let result: facebook_location_history::LocationHistory =
                 serde_json::from_str(&rawdata)?;
+            let total_records2 = total_records;
             let response = result.saveToDb(&conn)?;
-            println! ("({} records)", &result.location_history_v2.unwrap_or_default().len());
+            total_records= total_records + result.location_history.unwrap_or_default().len() + result.location_history_v2.unwrap_or_default().len();
+            println!(
+                "({} records)",
+                total_records-total_records2
```

Comment:
Can be refactored like below maybe...?

let total_records2 = total_records; // remove this line
let location_history_v1 = result.location_history.unwrap_or_default().len();
let location_history_v2 = result.location_history..unwrap_or_default().len();

println!("{} records: ", location_history_v1 + location_history_v2); // This output is as same as the result of `total_records= total_records + result.location_history.unwrap_or_default().len() + result.location_history_v2.unwrap_or_default().len();` ?

Reply:
![image](https://user-images.githubusercontent.com/79226149/121893169-f4382d00-cd1d-11eb-8d2e-86a3b6447e40.png)
It doesn't work.

Reply:
1. `..`unwrap_or_default → `.unwrap_or_default`

2. how about `location_history_v1 as u32 + location_history_v2 as u32`.

Reply:
The problem is that the `unwrap_or_default().len()` takes the ownership, so we can't use it multiple times but we need it twice (as on the following screenshot). That is the reason why I wrote the first version of the code as it was, even if it was redundant. 
![image](https://user-images.githubusercontent.com/79226149/121897502-bc7fb400-cd22-11eb-9d5a-9023925ef0c9.png)

Reply:
Ok, I understand how to change it. Thank you!